wg/iot-edge.md

```diff
@@ -0,0 +1,53 @@
+
+- **Educate and inform** users with unbiased information regarding edge computing related CNCF projects
+- **Strengthen the ecosystem** by fostering better collaboration between existing projects
+- **Curate best practices** related to the edge computing systems that are effective and actionable.
+- Identify gaps in the landscape and **attract new projects** to the ecosystem.
+- **Provide technical support** to users trying to implement edge computing systems
+
+## Scope
+The working group is interested in all topics related to building edge computing solutions, including:
+
+- Developing and adjusting **infrastructure for edge deployments**. As an extension to the cloud, edge computing infrastructure needs modifications to meet edge conditions. Resource constraints and intermittent network connectivity are commonly considered topics in Edge computing.
+- Development and configuration of **edge-specific workloads**. How do we write and deploy our workloads to edge environments? Topics like multi-architecture builds, multi-cluster deployments, access to system resources and GitOps are usually discussed in this regard.
+- Cloud-native **DevOps practices for edge** environments. How do we operate our workloads in edge deployment scenarios? Typical DevOps and Observability topics need to be adjusted to edge computing environments.
+
+## Deliverables
+The working group will build collaboration, knowledge, and projects in the edge computing space, including:
+
+- **Publications, presentations and whitepapers** on interesting topics - the group will try to stay ahead of the current technology trends and provide material on the latest innovative topics in the space.
+- **Example architectures and demos** - provide well documented use cases using existing projects and demos that can be used as starting point of future projects.
+- **Standards, tests and certifications** - the group will investigate if it is possible to certify solutions in well defined domains against known criteriums.
+- **Metrics** - provide guidelines for project planning in terms of resource usage.
+- **Ecosystem updates** - give updates about the landscape of cloud native projects on the edge at conferences, in publications, and to the Runtime TAG.
+
+## Current CNCF Edge-centric projects
+The following is a list of well-known CNCF projects used in edge-computing solutions:
+
+- Kubernetes
+- KubeEdge
+- Akri
+- K3s
+- WasmEdge
+
+The list is not exhaustive. Please submit a PR to add your edge-focused CNCF project or reach out to the #wg-iot-edge channel on Kubernetes Slack.
+
+## Operations
+### TOC Liaison
+- [Ricardo Aravena](https://github.com/raravena80), Rakuten 
```

Comment:
I believe that @raravena80 is our Runtime TAG liaison. @raravena80, what do you think is the best way to find a TOC liaison? Maybe we reach out to each of the Runtime TAG liaisons?

Reply:
Yep. I can be the WG's liaison for the TAG. For TOC liaisons you can include all or any of the TAG-Runtime [TOC liaisons](https://github.com/cncf/tag-runtime#toc-liaisons).   Can you change my employer to TruEra? Thx.

Reply:
Thanks Ricardo. I've updated the employer details.